### ISSUE TITLE
Can someone use yolov8 instead of yolov3 in detector?

### ISSUE THREAD

**Issue by lsm101010** [MVIG-SJTU/AlphaPose#1212]
Can someone use yolov8 instead of yolov3 in detector?

**Comment by Madou-AI** [comment]
i want to use the yolov9 but i don't know how

**Comment by lsm101010** [comment]
> i want to use the yolov9 but i don't know how

do you have any ideas?

**Comment by zhou-jered** [comment]
you only need to give the object's retangle, that's all you need

**Comment by valentin-fngr** [comment]
Yes but you will likely need to create a Yolov8 API under /detector. You can simply train a yolov8 from ultralytics, write a similar code as the yolo API in /detector/ and replace the model by the one you just trained.

**Comment by NigelHiggs30** [comment]
> you only need to give the object's retangle, that's all you need

This would be the easiest way to do this, does any currently know exactly to put this information in?
